index.html

```diff
@@ -0,0 +1,75 @@
+<!Doctype HTML>
+<html>
+  <head>
+    <title>Example</title>
+    <!-- Links CSS file -->
+    <link rel="stylesheet" href="index.css" type="text/css">
+
+    <!-- iphone & TV could have same amount of pixels.
+    So base content on width of device instead of pixels -->
+    <meta name="viewport" content="width=device-width">
+  </head>
+
+  <body>
+    <!-- Header section -->
+    <header class="header-container">
+      <h1>Reliable, efficient delivery</h1>
+      <h1>Powered by Technology</h1>
+
+      <p>Our Artificial Intelligence powered tools use milltions of

```

Comment:
Hello Callum, I think I spotted a type, should milltions be millions? Code looks good. Thank you for the helpful comments explaining the code.

Reply:
Awesome, thanks Kerry, only just seen your comment. I didn't spot this one, will get it resolved. 😃